### PR TITLE
Added $sample helper to the Aggregates builder

### PR DESCRIPTION
**Pull request by rozza**
JAVA-1937

### CODE REVIEW

File: driver-core/src/main/com/mongodb/client/model/Aggregates.java [hunk 1]
```diff
@@ -165,6 +165,19 @@ public static Bson out(final String collectionName) {
...
+     * Creates a $sample pipeline stage with the specified sample size
+     *
+     * @param size the sample size
+     * @return the $sample pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/sample/  $sample
+     * @mongodb.server.release 3.2
+     * @since 3.2
+     */
+    public static Bson sample(final int size) {
+        return new BsonDocument("$sample", new BsonDocument("size", new BsonInt32(size)));
```
Comment (evanchooly) [inline driver-core/src/main/com/mongodb/client/model/Aggregates.java]: You'll need to also add an option to allow disk use when this option is added.  I'm not sure if the API supports automatically adding it, though.  At the least, the docs should mention the potential need for this option.

Comment (rozza) [inline driver-core/src/main/com/mongodb/client/model/Aggregates.java]: Its really just to create the stage - so its not coupled to other stages.

Any information about disk use? http://docs.mongodb.org/manual/release-notes/3.1-dev-series-reference/#pipe._S_sample seems lacking on the subject - although the eventual manual docs should cover any caveats, which should be enough.

Comment (jyemin) [inline driver-core/src/main/com/mongodb/client/model/Aggregates.java]: Such an option already exists on AggregateIterable.  It wouldn't go here.

**Comment by jyemin** [comment]
lgtm

**Comment by rozza** [comment]
merging into local 3.2 branch

**Pull request closed without merging**